src/System.Text.Json/src/System/Text/Json/Node/JsonObjectProperty.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    internal class JsonObjectProperty
+    {
+        internal string _name { get; set; }
+        internal JsonNode _value { get; set; }
+        internal JsonObjectProperty _prev { get; set; }
+        internal JsonObjectProperty _next { get; set; }
```

Comment:
These are named like fields, but are properties.  PascalCase all of them, please.